### PR TITLE
OCPBUGS-1772: Pre-caching feature blocks CGU CR

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.11

Issue: https://issues.redhat.com/browse/OCPBUGS-1772

Link to docs preview:
Last entry in known issues: https://51022--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-11-release-notes.html#ocp-4-11-6

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by opayne1** [comment]
@amolnar-rh Is this a KI that should have went in yesterday and we missed it, or is this a last minute one that came up? Either way, I think this may need to go through change management since 4.11.6 is already live.

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -2845,6 +2845,8 @@ $ oc annotate pod dns-default target.workload.openshift.io/management='{"effect"
 +
 (link:https://issues.redhat.com/browse/OCPBUGS-753[*OCPBUGSM-753*])
 
+* Enabling the pre-caching feature of {cgu-operator-full} blocks the `ClusterGroupUpgrades` CR. To workaround this issue, do not enable the pre-caching feature. (link:https://issues.redhat.com/browse/OCPBUGS-1772[*OCPBUGS-1772*])
```
Comment (opayne1) [inline release_notes/ocp-4-11-release-notes.adoc:2848]:
```suggestion
* Enabling the pre-caching feature of {cgu-operator-full} blocks the `ClusterGroupUpgrades` custom resource (CR). As a workaround, do not enable the pre-caching feature. (link:https://issues.redhat.com/browse/OCPBUGS-1772[*OCPBUGS-1772*])
```

**Review by opayne1 — COMMENTED** [review]
Just a small comment on the text as well.

File: release_notes/ocp-4-11-release-notes.adoc (hunk 1 above)
Comment (amolnar-rh) [reply release_notes/ocp-4-11-release-notes.adoc]: @opayne1 Thank you! Updated

**Review by imiller0 — COMMENTED** [review]
LGTM

**Comment by amolnar-rh** [comment]
The fix has been shipped, release note is no longer needed: https://issues.redhat.com/browse/OCPBUGS-1772

**Pull request closed without merging**